README.md

```diff
@@ -111,15 +111,34 @@ public class ExampleApplication extends Application {
 ```
 
 <ol start="4">
-  <li>Call the state tracker's <code>promptIfReady</code> method when appropriate, passing in the current <code>Activity</code> and your <code>DefaultLayoutPromptView</code> instance:</li>
+  <li>Get the shared <code>Amplify</code> instance and call its <code>promptIfReady</code> method when appropriate, passing in your <code>DefaultLayoutPromptView</code> instance:</li>
 </ol>
 
 ```java
-DefaultLayoutPromptView promptView = (DefaultLayoutPromptView) findViewById(R.id.prompt_view);
-Amplify.get(context).promptIfReady(activity, promptView);
+public class ExampleActivity extends Activity {
+
+    @Override
+    public void onCreate(@Nullable final Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_example);
+
+        /*
+         * Be careful: don't re-prompt after a configuration change!
```

Comment:
If you do, you'll count too many prompt-related events, which will affect your prompt's behavior if you have applied any rules based on these events!